addons/udes_stock/models/stock_picking.py

```diff
@@ -1863,14 +1885,25 @@ def _get_suggested_location_by_products(self, move_line_ids, products=None):
         if not products:
             raise ValidationError(_("Products missing to suggest location for."))
```

Comment:
Nit: Error message reads a bit oddly to me. Maybe "No products to suggestion location for." instead? Feel free to resolve though, probably just me. :)

Reply:
I'm going to leave this one as it is as it was a pre-existing error message